src/main/resources/application-dev.yml

```
@@ -0,0 +1,23 @@
+  jpa:
+    hibernate:
+      ddl-auto: update
+
+    properties:
+      hibernate:
+        format_sql: true
+        show_sql: true
+        database-platform: org.hibernate.dialect.PostgreSQLDialect
+
+jwt:
+  secret: ${JWT_SECRET_FOR_DEV}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)